### PR TITLE
bpo-29920: Document cgitb.text and cgitb.html

### PR DESCRIPTION
**Pull request by masklinn**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://cpython-devguide.readthedocs.io/pullrequest.html#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Doc/library/cgitb.rst [hunk 1]
```diff
@@ -54,6 +54,20 @@ analysis.
    value forces plain text output.  The default value is ``"html"``.
 
 
+.. function:: text(info[, context=5])
+
+   Handles the exception described by *info* (a 3-tuple containing the result of
```
Comment (marco-buttu) [inline Doc/library/cgitb.rst]: You should write _This function handles_, as in the rest of the file.

File: Doc/library/cgitb.rst [hunk 2]
```diff
@@ -54,6 +54,20 @@ analysis.
...
+   :func:`sys.exc_info`), formatting its traceback as text and returning the
+   result as a string.
+
+
+.. function:: html(info[, context=5])
+
+   Handles the exception described by *info* (a 3-tuple containing the result of
+   :func:`sys.exc_info`), formatting its traceback as html and returning the
```
Comment (marco-buttu) [inline Doc/library/cgitb.rst]: It should be `HTML` (uppercase), as in the rest of the file.  It is lowercase only for the string argument.

File: Doc/library/cgitb.rst [hunk 3]
```diff
@@ -54,6 +54,20 @@ analysis.
    value forces plain text output.  The default value is ``"html"``.
 
 
+.. function:: text(info[, context=5])
```
Comment (vadmium) [inline Doc/library/cgitb.rst]: The square bracket notation conflicts with the notation in the rest of the page

Comment (berkerpeksag) [reply Doc/library/cgitb.rst]: Martin is correct. The signature should read `text(info, context=5)`.

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

File: Doc/library/cgitb.rst (hunk 1 above)
Comment (masklinn) [reply Doc/library/cgitb.rst]: Intro fixed.

File: Doc/library/cgitb.rst (hunk 2 above)
Comment (masklinn) [reply Doc/library/cgitb.rst]: Casing fixed.

File: Doc/library/cgitb.rst (hunk 3 above)
Comment (masklinn) [reply Doc/library/cgitb.rst]: Argspec fixed.

**Comment by masklinn** [comment]
PR updated to take reviews in account. I also augmented the text slightly to actually document the *context* parameter rather than have readers need to check the description of other functions.

**Review by berkerpeksag — APPROVED** [review]

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**